tests/Form/SelectTest.php

```diff
@@ -63,7 +70,8 @@ public function testElementSelectedStateCheck($selectName, $optionValue, $option
         $session->visit($this->pathTo('/multiselect_form.html'));
         $select = $webAssert->fieldExists($selectName);
 
-        $option = $webAssert->elementExists('named', array('option', $optionValue));
+        $option = $select->find('named', array('option', $optionValue));
```

Comment:
This resolves a nasty bug. We were searching for the option with value `$optionValue` in the whole page, rather than in the scope of the select.

Reply:
In fact you can return old code version, but specify `$select` in 3rd parameter of `$webAssert->elementExists` method call. This will reduce search scope to that SELECT only.

Reply:
Whoops. Missed that. Thanks.

Reply:
@aik099 Updated.